src/main/java/org/xstefank/model/yaml/Format.java

```diff
@@ -16,19 +17,15 @@ public void setTitle(String title) {
         this.title = title;
     }
 
-    public Description getDescription() {
-        return description;
-    }
+    public String getCommit() { return commit; }
 
-    public void setDescription(Description description) {
-        this.description = description;
-    }
+    public void setCommit(String commit) { this.commit = commit; }
 
-    public List<String> getAdditional() {
-        return additional;
-    }
+    public Description getDescription() { return description; }
 
-    public void setAdditional(List<String> additional) {
```

Comment:
In general, try to avoid changing code that is not a part of the fix for a given issue. I understand that this code is not probably being used anywhere so IDE is doing its magic, but try to avoid it. This should be a separate fix for cleaning dead code.